astaire/astaire.supervisord.conf

```diff
@@ -0,0 +1,5 @@
+[program:astaire]
+command=/etc/init.d/astaire run
+startretries=100
```

Comment:
Is there a reason we have startretries=100 here and not for most of the others? Should we have it on all of them, or is this astaire-specific?